qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -116,39 +136,59 @@ public CompletableFuture<List<CodeAction>> doCodeActions(Template template, Code
 				QuteErrorCode errorCode = QuteErrorCode.getErrorCode(diagnostic.getCode());
 				if (errorCode != null) {
 					switch (errorCode) {
-						case UndefinedObject:
-							// The following Qute template:
-							// {undefinedObject}
-							//
-							// will provide a quickfix like:
-							//
-							// Declare `undefinedObject` with parameter declaration."
-							doCodeActionsForUndefinedObject(template, diagnostic, errorCode, codeActions);
-							break;
-						case UndefinedSectionTag:
-							// The following Qute template:
-							// {#undefinedTag }
-							//
-							// will provide a quickfix like:
-							//
-							// Create `undefinedTag`"
-							doCodeActionsForUndefinedSectionTag(template, diagnostic, codeActions);
-							break;
-						case UndefinedNamespace:
-							// The following Qute template:
-							// {undefinedNamespace:xyz}
-							doCodeActionToSetIgnoreSeverity(template, Collections.singletonList(diagnostic), errorCode,
-									codeActions, UNDEFINED_NAMESPACE_SEVERITY_SETTING);
-							break;
-						case UnknownProperty:
-							doCodeActionToCreateProperty(template, diagnostic, errorCode, codeActions,
-									codeActionResolveFutures,
-									resolver,
-									sharedSettings);
-							break;
-						default:
-							break;
+					case UndefinedObject:
+						// The following Qute template:
+						// {undefinedObject}
+						//
+						// will provide a quickfix like:
+						//
+						// Declare `undefinedObject` with parameter declaration."
+						doCodeActionsForUndefinedObject(template, diagnostic, errorCode, codeActions);
+						break;
+					case UndefinedSectionTag:
+						// The following Qute template:
+						// {#undefinedTag }
+						//
+						// will provide a quickfix like:
+						//
+						// Create `undefinedTag`"
+						doCodeActionsForUndefinedSectionTag(template, diagnostic, codeActions);
+						break;
+					case UndefinedNamespace:
+						// The following Qute template:
+						// {undefinedNamespace:xyz}
+						doCodeActionToSetIgnoreSeverity(template, Collections.singletonList(diagnostic), errorCode,
+								codeActions, UNDEFINED_NAMESPACE_SEVERITY_SETTING);
+						break;
+					case UnknownMethod:
+						// CodeAction(s) to replace text with similar suggestions
+						try {

```

Comment:
To be consistent with other error code, I will create doCodeActionsForUnknownMethod and move this code in the doCodeActionsForUnknownMethod .

Reply:
Please remove this code which support similar value for method. We will see that in an another PR.

Reply:
I managed to add support for `UnknownMethod`:
![Screenshot from 2022-07-28 13-46-08](https://user-images.githubusercontent.com/26389510/181603764-81519e5a-3f17-46da-a078-1a548062d9f7.png)

Reply:
OK but it will take more time to review it because your  code that I read can throw some NPE and it misses some filter. I would prefer that you work on tests but as you wish.